configure.ac

```diff
@@ -66,6 +66,16 @@ AC_ARG_ENABLE([extratests],
 	AS_HELP_STRING([--enable-extratests], [Enable extra tests that may be unreliable on some platforms]))
 AM_CONDITIONAL([ENABLE_EXTRATESTS], [test "x$enable_extratests" = xyes])
 
+AC_ARG_ENABLE([tests],
+       [AS_HELP_STRING([--disable-tests], [Enable tests (default: yes)])],
+       [if test "x$enable-tests" = "xyes"; then
```

Comment:
s/enable-tests/enable_tests/
This causes "--enable-tests", "--enable-tests=yes", "--enable-tests=no" gives disabling tests.

Reply:
And I wonder if we could do this only with "enable_tests" without "have_tests" to keep simple.